src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Sql.tests.ps1

```diff
@@ -99,7 +99,7 @@ InModuleScope Arcus.Scripting.Sql {
             # avoid having timeout errors during the test themselves.
             try {
                 Write-Host "Execute dummy SQL statement to make sure the Azure SQL DB is resumed."
-                Invoke-Sqlcmd @params -Query "SELECT TOP 1 FROM INFORMATION_SCHEMA.TABLES" -ConnectionTimeout 60 -Verbose
+                Invoke-Sqlcmd @params -Query "SELECT TOP 1 FROM INFORMATION_SCHEMA.TABLES" -ConnectionTimeout 60 -Verbose -ErrorAction SilentlyContinue
```

Comment:
Do we still need the try/catch statement if we specify `SilentlyContinue`?

Reply:
Probably not 😉 .

Reply:
Changed it in https://github.com/arcus-azure/arcus.scripting/pull/233/commits/fb24a476f1cdc49a93579b17558ce08eeffadc82

Reply:
Had to fix the SQL statement, as there was missing a selection.